simulator/bpu/bpentry.h

```diff
@@ -79,9 +79,10 @@ class BPEntryOneBit final : public BPEntry
             /* reset */
             state = default_state;
             _target = target;
-        }
-
-        state = static_cast<State>( is_taken);
+        } else 
+	{ 
```

Comment:
We do not use tabs for identation. Please check this manual: https://github.com/MIPT-ILab/mipt-mips/wiki/Code-guidelines#indentation.